### PR TITLE
skip ordinal bin bucket sorting if no configuration object is provided

### PR DESCRIPTION
**Pull request by jthrilly**
Mirrors the fix provided here: #525 

This component should perhaps be refactored as a HOC of node list, but for now this will suffice.

### CODE REVIEW

File: src/containers/Interfaces/OrdinalBin.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ class OrdinalBin extends Component {
             id={'NODE_BUCKET'}
             itemType="EXISTING_NODE"
             onDrop={this.onDrop}
-            sortOptions={prompt.bucketSortOrder}
+            sortOOrder={prompt.bucketSortOrder}
```
Comment (bryfox) [inline src/containers/Interfaces/OrdinalBin.js]: Typo here

**Comment by bryfox** [comment]
Got my branches mixed up; UI ref update matches the new-start-screen changes. 👍

File: src/containers/Interfaces/OrdinalBin.js (hunk 1 above)
Comment (jthrilly) [reply src/containers/Interfaces/OrdinalBin.js]: Ha! I think this is literally down to the train ride being bumpy.

**Comment by bryfox** [comment]
I'm going to close this & update ordinal bin as part of #530.

**Pull request closed without merging**